docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +12,49 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-1]]
+=== Logstash 6.1.1 Release Notes
+*  There are no user-facing changes in Logstash core in this release.
+
+[float]
+==== Input Plugins
+
+*`Beats`*:
+
+* Fix issue with close_wait connections, making sure keep alive sent back to the client. (https://github.com/logstash-plugins/logstash-input-beats/pull/272[#272])
+
+*`HTTP`*:
+
+* In the event that all webserver threads are busy this plugin will now return a 429, busy, error.
```

Comment:
since we're linking prs/issues in other entries, we can also add https://github.com/logstash-plugins/logstash-input-http/pull/75 here